docs/src/pages/_app.page.tsx

```diff
@@ -1,7 +1,5 @@
 import Head from 'next/head';
-import '../styles/styles.css';
-import '@aws-amplify/ui-react/styles.css';
-import '../styles/primitivesStyles.css';
+import '../styles/index.css';
```

Comment:
Moving CSS imports to a single `index.css` file so that we can control the order of styles. At least in development NextJS was not importing these CSS files in the right order with the `primitiveStyles.css` being loaded before `@aws-amplify/ui-react/styles.css`, which blew up those CSS overrides for examples where we add a className to a primitive to style it.